### PR TITLE
added screeshots

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user feedback #10444 , so i added new screenshots which is suitable for windows 11

### CODE REVIEW

File: windows/client-management/administrative-tools-in-windows-10.md [hunk 1]
```diff
@@ -15,21 +15,29 @@ ms.topic: article
...
-The tools in the folder might vary depending on which edition of Windows you are using. 
+The tools in the folder might vary depending on which edition of Windows 10 you are using. 
+
+![Screenshot of folder of administrative tools.](images/admin-tools-folder.png)
+
+Windows Tools (Windows 11) is a folder in Control Panel that contains tools for system administrators and advanced users. 
+
+![Screenshot of Control Panel.](https://user-images.githubusercontent.com/3296790/159910156-954c4835-a4a0-4dd3-8858-8f7796ae8ad9.png)
+
+The tools in the folder might vary depending on which edition of Windows 10 you are using.
```
Comment (JohanFreelancer9) [inline windows/client-management/administrative-tools-in-windows-10.md:38]:
```suggestion
The tools in the folder might vary depending on which edition of Windows 10 you are using. 
```
```suggestion
The tools in the folder might vary depending on which edition of Windows 11 you are using. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RAJU2529, kindly commit the suggested change if you agree with it. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — CHANGES_REQUESTED** [review]
Thanks @RAJU2529 for the contribution, and thanks @JohanFreelancer9 for the review.
There are some other aspects here that I'd like to address before finalizing. The article still includes "10" in the table of contents. And we should be storing images with the markdown in our repository, not in the GitHub user content location.
I'll make some changes into this PR soon - no changes needed from you at this time.

**Review by aczechowski — APPROVED** [review]

**Comment by aczechowski** [comment]
@RAJU2529 I'm going to merge my changes, they should publish live on Tuesday 3/29 around 11 AM Pacific. If my changes don't meet your intent, please open another pull request or file feedback on the article. Thanks for your contribution to raise this issue!

**Pull request merged**